queue/queue.py

```diff
@@ -1,18 +1,94 @@
-from doubly_linked_list import ListNode
-from doubly_linked_list import DoublyLinkedList
+"""Each ListNode holds a reference to its previous node
+as well as its next node in the List."""
+class ListNode:
+    def __init__(self, value, prev=None, next=None):
+        self.value = value
+        self.prev = prev
+        self.next = next
+
+class Linked_lists:
+    def _init_(self, node=None):
+        self.head = node
+        self.tail = node
+        self.length = 0
+
+    #Remove nodes from head
+    def remove_from_head(self):
+        # return None if there is no head (i.e. the list is empty)
+        if not self.head:
+            return None
+        # if head has no next, then we have a single element in our list
+        if not self.head.get_next():
```

Comment:
this is for the case where there's only one element :) you can also check self.length, or if `self.head == self.tail`. 
Great job!